contribution/contributions.md

```diff
@@ -146,7 +146,12 @@ We are following Google Code style:
 * [Java style](https://google.github.io/styleguide/javaguide.html)
 * [Shell style](https://google.github.io/styleguide/shell.xml)
 
-Check style report location are in `${submodule}/target/site/checkstyle.html`
+There are some plugins to format, lint your code in IDE (use [_tools/checkstyle.xml](https://github.com/apache/zeppelin/tree/master/_tools) as rules)
+
+* [Checkstyle plugin for Intellij](http://eclipse-cs.sourceforge.net/#!/project-setup)
+* [Checkstyle plugin for Eclipse](http://stackoverflow.com/questions/26955766/intellij-idea-checkstyle)
+
+Checkstyle report location are in `${submodule}/target/site/checkstyle.html`
 Test coverage report location are in `${submodule}/target/site/cobertura/index.html`
 
```

Comment:
Seems there are some minor grammar errors are in here. Let's fix them using this PR :D

"The location of test coverage report is `${submodule}/target/site/cobertura/index.html`." 
"The location of checkstyle report is `${submodule}/target/site/checkstyle.html`."

How about this?